README.md

```diff
@@ -72,6 +73,11 @@ It's 100% Open Source and licensed under the [APACHE2](LICENSE).
     posse --help for help
 ```
 
+### Delete all the VPCs in an AWS Account
```

Comment:
```suggestion
### Delete all the VPCs in an AWS Account

Best-practices for AWS call for not using the default VPC, but rather, creating a new set of VPCs as necessary. AWS Security Hub will flag the default VPCs as non-compliant if they aren't configured with best-practices. Rather than jumping through hoops, it's easier to delete to default VPCs. This task cannot be accomplished with terraform, so this command is necessary.
```

Reply:
310128dfdd35e93edaed0e9e18af8da294e32d70